src/Desktop.Analyzers/Core/DoNotUseInsecureCryptographicAlgorithms.cs

```diff
@@ -55,6 +55,11 @@ private static DiagnosticDescriptor CreateDiagnosticDescriptor(string ruleId, Lo
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.EnableConcurrentExecution();
+
+            // Security analyzer - analyze and report diagnostics in generated code.
```

Comment:
End user will probably suppress the diagnostic as it is in generated code, but will most likely report it to the tool author to fix their code generator.